specs/guidelines.md

```diff
@@ -84,7 +84,23 @@ the latter would be the creation date of the (digital) entity representing the w
 that this is the actual date at which the measured value was obtained (from a sensor, by visual observation, etc.), and that
 date might be different than the date (metadata attribute named `dateModified` as per NGSIv2) at which the attribute
 of the digital entity was updated, as typically there might be delay,
-specially on IoT networks which deliver data only at specific timeslots. 
+specially on IoT networks which deliver data only at specific timeslots.
+
+## Internationalization (i18N)
+
+There can be certain entity attributes which content is subject to be internationalized. For instance, the description of a Point of Interest.
+The i18N guidelines for the FIWARE Data Models are as follows:
+
++ By default the value of an attribute subject to be internationalized shall be expressed in *British English* (`en-GB`).
++ For each language variant available for an attribute subject to be internationalized there shall be an additional
+entity attribute which name shall be in the form:
+
+`<Attribute_Name>_<Language_Tag>`  where `Language_Tag` shall be a language tag as mandated by [RFC 5646](https://www.rfc-editor.org/rfc/rfc5646.txt).
+W3C provides guidelines on [how to use language tags](https://www.w3.org/International/articles/language-tags/).  
+
+Example:
+
+an attribute named `description_es` can be used to convey the value of a `description` attribute in Spanish. 
```

Comment:
How can I tell which languages are supported for each attribute if all variations are top level?

e.g.

```json
{
   "description":  "Something Colorful in English",
   "description_en-gb":  "Something Colourful in British English",
   "description_de-de":  "Etwas Buntes auf Englisch ",
   "description_extended" : "The Color Blue"
}
```

However `description_extended` is **NOT** an exonym, but another field - How can I identify that?

Maybe we should do something like `name`/`alternativeName` as defined in schema.org:

```json
{
   "description":  "Something Colorful in English",
   "alternativeDescription" : [
       {"en-gb":  "Something Colourful in British English"},
       {"de-de":  "Etwas Buntes auf Englisch "}
    ],
   "description_extended" : "The Color Blue"
}
```

Reply:
the JSON-LD @context allows you to fix the issue you are mentioning, look at

https://www.w3.org/TR/json-ld/#string-internationalization 

essentially, more than one JSON term could refer to the same property at semantic level and could be tagged with a language name at @context level as well. 

On the other hand the application shall try to detect suffixes which are actually valid language tags ... and not try to detect only attribute prefixes ...

Reply:
OK, then could we clarify the usage by linking to the json-ld spec? Or even better, copy a relevant example:

```
{
  "@context": {
    ...
    "ex": "http://example.com/vocab/",
    "@language": "ja",
    "name": { "@id": "ex:name", "@language": null },
    "occupation": { "@id": "ex:occupation" },
    "occupation_en": { "@id": "ex:occupation", "@language": "en" },
    "occupation_cs": { "@id": "ex:occupation", "@language": "cs" }
  },
  "name": "Yagyū Muneyoshi",
  "occupation": "忍者",
  "occupation_en": "Ninja",
  "occupation_cs": "Nindža",
  ...
}
```

I assume you could use the Paris exonyms for this if you like.